packages/transition-frontend/package.json

```diff
@@ -117,6 +117,6 @@
     "webpack": "^4.43.0",
     "webpack-cdn-plugin": "^3.3.1",
     "webpack-cli": "^3.3.12",
-    "webpack-glsl-loader": "^1.0.1"
+    "@davcri/webpack-glsl-loader": "^1.0.2"
```

Comment:
Why `@davcri` loader? It has only 7 weekly downloads. Would https://www.npmjs.com/package/ts-shader-loader do the same job and fix the issue this fixes?

Reply:
J'ai regardé le code, rien de louche. C'est un fork à partir de <https://github.com/grieve/webpack-glsl-loader/issues/9>

<https://www.npmjs.com/package/ts-shader-loader> pourrait être une bonne option, j'ai pas trop d'opinion, j'ai pas d'expérience avec ```glsl```.